tests/unit/Codeception/Module/RestTest.php

```diff
@@ -442,6 +442,63 @@ public function testAmDigestAuthenticatedThrowsExceptionWithFunctionalModules()
         $this->module->amDigestAuthenticated('username', 'password');
     }
 
+    /**
+     * @param $configUrl
+     * @param $requestUrl
+     * @param $expectedFullUrl
+     *
+     * @dataProvider configAndRequestUrls
+     */
+    public function testRestExecute($configUrl, $requestUrl, $expectedFullUrl)
+    {
+        $connectionModule = $this->createMock(
+            \Codeception\Module\UniversalFramework::class
+        );
+        $connectionModule
+            ->expects($this->once())
+            ->method('_request')
+            ->will(
+                $this->returnCallback(function($method,
+                    $uri,
+                    $parameters,
+                    $files,
+                    $server,
+                    $content
+                ) use ($expectedFullUrl) {
+                    \PHPUnit\Framework\Assert::assertEquals($expectedFullUrl, $uri);
+                })
+            );
+
+        $config = ['url' => $configUrl];
+
+        /** @var \Codeception\Module\REST */
+        $module = Stub::make('\Codeception\Module\REST');
+        $module->_setConfig($config);
+        $module->_inject($connectionModule);
+        $module->_initialize();
+        $module->_before(Stub::makeEmpty('\Codeception\Test\Test'));
+
+        $module->sendGET($requestUrl);
+    }
+
+    public static function configAndRequestUrls()
+    {
+        return array(
```

Comment:
Please use the short array syntax - `[]`.

Reply:
yep, will do that though it wouldn't change much as 

codeception on bug-5473-rest-module-doubles-leading-slashes [$]
$ grep -RIc ' array(' src/Codeception/ | wc -l
     269

codeception on bug-5473-rest-module-doubles-leading-slashes [$]
$ grep -RIc ' array(' tests/ | wc -l
     646